contents.json

```diff
@@ -6321,6 +6321,12 @@
       "description": "Highly customizable tab bar for SwiftUI applications.",
       "homepage": "https://github.com/onl1ner/TabBar",
       "tags": ["swift", "swiftui", "tab-bar", "iOS"]
+    },{
+      "title": "GoodNetworking",
+      "category": "network",
+      "description": "📡 GoodNetworking is an iOS library written in Swift that simplifies HTTP networking",
```

Comment:
Hi @andrej-jasso ☺️ Could you update it a bit considering [the following](https://github.com/matteocrippa/awesome-swift/blob/master/.github/CONTRIBUTING.md): 
- The description usually should not include "written in Swift" or "in Swift" as everything on the list would have that
- Start the description with a capital and end with a full stop/period

I would write it as: "📡 GoodNetworking is a library that simplifies HTTP networking."

Reply:
Sorry, thought it applied to PR description only :)